front/components/navigation/Navigation.tsx

```diff
@@ -18,6 +18,8 @@ interface NavigationProps {
   subscription: SubscriptionType;
   navChildren?: React.ReactNode;
   subNavigation?: SidebarNavigation[] | null;
+  isNavigationBarOpen: boolean;
```

Comment:
Why not using the Sidebar context for this?

Reply:
Sidebar and navigation bar are not the same thing. Sidebar context doesn't do anything for the navigation bar

Reply:
Interesseting 🤯

Reply:
Ok Indeed the context is only used on Mobile 🤯